tests/integration/golang/aim/app/get_app_test.go

```diff
@@ -0,0 +1,92 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/database"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetAppTestSuite struct {
+	suite.Suite
+	client      *helpers.HttpClient
+	appFixtures *fixtures.AppFixtures
+	apps        []*database.App
+}
+
+func TestGetAppTestSuite(t *testing.T) {
+	suite.Run(t, new(GetAppTestSuite))
+}
+
+func (s *GetAppTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(os.Getenv("SERVICE_BASE_URL"))
+
+	appFixtures, err := fixtures.NewAppFixtures(os.Getenv("DATABASE_DSN"))
+	assert.Nil(s.T(), err)
+	s.appFixtures = appFixtures
+
+	s.apps, err = s.appFixtures.CreateApps(context.Background(), 1)
+	assert.Nil(s.T(), err)
+}
+
+func (s *GetAppTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.appFixtures.UnloadFixtures())
+	}()
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "GetAppWithExistingID",
```

Comment:
i think it's prudent to setup for table test in all cases -- the occasional test which is not table driven is hard to read

Reply:
emmm, but why? this task for example -> https://github.com/G-Research/fasttrackml/blob/d1c362fca156b8069bfa61f1e3f43a54eae5782b/tests/integration/golang/mlflow/experiment/create_test.go#L37 looks pretty clean and understandable. I just want to say, that can we have setup for each test as small as possible? If we test only one case - just usual plain test. if we need to test only one entity - just create one entity instead of entities set. this is all adds purity to the test and understanding.

Reply:
Table-driven tests can explain what you're testing and looking for, whereas if there's not a table, "Test_Ok" and a collection of assertions are the only description which is not descriptive

Reply:
but I will make the change